### PR TITLE
Add debugging option for ROS launch files

### PR DESCRIPTION
**Pull request by DaniGarciaLopez**
Thanks @athackst for this template, it made my workflow much more efficient!

I wanted to contribute to this repository by adding the ability to debug ROS launch files directly. Sometimes you need to debug a node (or several) that receives a lot of configuration parameters from a launch file and [parsing them manually](https://github.com/athackst/vscode_ros2_workspace/issues/10#issuecomment-946364080) might not be the best solution.

This uses the official Microsoft ROS extension for VS Code ([ms-iot.vscode-ros](https://marketplace.visualstudio.com/items?itemName=ms-iot.vscode-ros)). More info [here](https://marketplace.visualstudio.com/items?itemName=ms-iot.vscode-ros).

launch-and-debug-nodes

### CODE REVIEW

**Comment by athackst** [comment]
Neat!

File: .vscode/launch.json [hunk 1]
```diff
@@ -50,7 +50,15 @@
...
       ]
-    }
+    },
+    //Example of a ROS Launch file
+    {
+      "name": "ROS: Launch File",
+      "type": "ros",
+      "request": "launch",
+      "preLaunchTask": "build",
+      "target": "${workspaceFolder}/install/share/${input:package}/launch/${input:ros_launch}",
```
Comment (athackst) [inline .vscode/launch.json:60]: Does this need a merge install and isolated install entry?

Comment (DaniGarciaLopez) [reply .vscode/launch.json:60]: Good catch! Added the entry for isolated install

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
 				"twxs.cmake",
 				"yzhang.markdown-all-in-one",
 				"zachflower.uncrustify",
-				"betwo.b2-catkin-tools"
+				"betwo.b2-catkin-tools",
+				"ms-iot.vscode-ros"
```
Comment (athackst) [inline .devcontainer/devcontainer.json:42]: nit: put this in alphabetical order

**Review by athackst — APPROVED** [review]
One small nit, otherwise LGTM

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (DaniGarciaLopez) [reply .devcontainer/devcontainer.json]: Done!

**Pull request merged**